packages/st2/debian/rules

```diff
@@ -6,29 +6,38 @@
 #include /usr/share/dpkg/default.mk
 
 # Virtualenv and pip should be updated (better not to use the distro defaults).
-# Updated ones can be found under /use/local.
+# Updated ones can be found under /usr/local.
 PATH = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 WHEELDIR ?= /tmp/wheelhouse
 DH_VIRTUALENV_INSTALL_ROOT := /opt/stackstorm
 export DH_VIRTUALENV_INSTALL_ROOT
 
 IS_SYSTEMD = $(shell command -v dh_systemd_enable > /dev/null 2>&1 && echo true)
 DEB_DISTRO := $(shell lsb_release -cs)
+ifneq (,$(wildcard /etc/centos-release))
+   EL_DISTRO := centos
+   EL_VERSION := $(shell cat /etc/centos-release | grep -oP '(?<= )[0-9]+(?=\.)')
```

Comment:
Looks like this file was using tabs instead of spaces as indentation. Would be better to stick with original format so we don't have spaces vs tabs mixed which would only add confusion for someone in future.